pytest/tests/sanity/state_sync4.py

```diff
@@ -18,8 +20,23 @@
 MAX_SYNC_WAIT = 30
 EPOCH_LENGTH = 10
 
+state_parts_dir = str(pathlib.Path(tempfile.gettempdir()) / 'state_parts')
+
 node0_config = {
-    "state_sync_enabled": True,
+    "state_sync": {
+        "dump": {
+            "location": {
+                "Filesystem": {
+                    "root_dir": state_parts_dir
+                }
+            },
```

Comment:
mini nit 2 - maybe even put it in the testing utilities so that you can reuse across multiple tests